packager/media/codecs/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Bitmovin Inc. All rights reserved.
```

Comment:
Please put "Google LLC" in the copyright header.  This is consistent with Google's requirement that contributors sign a CLA.

(I don't make these rules.)

Reply:
No problem. Can I add "Google LLC" _in addition_ to Bitmovin or must I replace it?